contracts/lib/Organization.sol

```diff
@@ -260,4 +314,39 @@ contract Organization is OrganizationInterface, IsMemberInterface {
         isWorker_ = workers[_worker] > block.number;
     }
 
+
+    /* Private functions */
+
+    /**
+     * @notice Sets worker and its expiration block height.
```

Comment:
Let's also mention, that if worker exists, the function updates it's data (expiration height in this case).

Reply:
👍 done